src/reducers/dragInProgress.js

```diff
@@ -1,3 +1,4 @@
-export default (state, { value }) => ({
-  dragInProgress: value
+export default (state, { value, draggedThoughtsRanked }) => ({
+  dragInProgress: value,
+  draggedThoughtsRanked: state.draggedThoughtsRanked ? (!draggedThoughtsRanked ? draggedThoughtsRanked : state.draggedThoughtsRanked) : draggedThoughtsRanked
```

Comment:
Can you write a comment that helps explain the logic here?

I tried multiple times to simplify a statement of this form but I'm not sure its reducible!

```js
a
  ? (!b ? null : a)
  : b
```